src/librbd/internal.cc

```diff
@@ -2217,6 +2218,15 @@ int mirror_image_disable_internal(ImageCtx *ictx, bool force,
         return -EBUSY;
       }
 
+      cls::rbd::GroupSpec s;
+      r = cls_client::image_get_group(&io_ctx, header_oid, s);
+      if ((!s.group_id.empty()) && (s.pool_id != -1)) {
```

Comment:
Minor: could add a "is_valid" method to GroupSpec to hide the internal details of a valid vs invalid spec.